src/components/tooltip/Tooltip.scss

```diff
@@ -88,6 +93,23 @@ $tooltip-offset: 4px; /* amount of space between target and tooltip arrow */
         }
     }
 
+    .tooltip-callout {
+        display: flex;
+        flex-flow: column;
+        font-weight: normal;
+        font-size: 13px;
+        line-height: 1.54;
```

Comment:
Can we use a more standardized value to avoid this magic number?

Reply:
I copied the styles from left-nav-callout css style. I'm thinking at some point we can replace left-nav-callout with this more generic class tooltip-callout. If I remove it it looks more condensed than the given style. And If i increase it, it looks more spread apart. I think I will keep it as close to design as possible and consistent with the left-nav-callout unless you think it's super important and should be changed

Reply:
1.54 * 13px = 20px, which was probably meant to be the height of the element